framework/packages/abstract-client/src/client.rs

```diff
@@ -19,6 +21,20 @@ impl<Chain: CwEnv> AbstractClient<Chain> {
         Ok(Self { abstr })
     }
 
+    pub fn name_service(&self) -> &AnsHost<Chain> {
+        &self.abstr.ans_host
+    }
+
+    pub fn version_control(&self) -> &VersionControl<Chain> {
+        &self.abstr.version_control
+    }
+
+    // Is there a way for us to return `AbstractClientResult<BlockInfo>` here instead? I think we
+    // would need to add <T: CwEnv> to `AbstractClientError` which I am not sure we want to do.
+    pub fn block_info(&self) -> Result<BlockInfo, <Chain as cw_orch::prelude::TxHandler>::Error> {
+        self.environment().block_info()
```

Comment:
This error type supports `Into<CwEnvError>` which is then convertable to the client error